virttest/qemu_storage.py

```diff
@@ -986,9 +986,12 @@ def convert(self, params, root_dir, cache_mode=None,
         convert_cmd = self.image_cmd + " " + \
             self._cmd_formatter.format(self.convert_cmd, **cmd_dict)
 
-        LOG.info("Convert image %s from %s to %s", self.image_filename,
-                 self.image_format, convert_image.image_format)
-        process.run(convert_cmd)
+        timeout = params.get_numeric("conversion_timeout", 12000)
```

Comment:
Suggest the param name `image_conversion_timeout` to avoid potential naming collisions.